package.json

```diff
@@ -4,9 +4,13 @@
 	"description": "Testing Sprint Challenge",
 	"main": "index.js",
 	"scripts": {
+		"server": "nodemon index.js",
```

Comment:
without using the cross-env and passing this test script the testing env this is just running on the regular database. you can achieve this through this script as your test script when you bring in the package cross-env to your project   "test": "cross-env DB_ENV=testing jest --watch --verbose",